cloudbuild/nightly.yaml

```diff
@@ -0,0 +1,52 @@
+steps:
+  # 1. Create a Docker image containing hadoop-connectors repo
+  - name: 'gcr.io/cloud-builders/docker'
+    id: 'docker-build'
+    args: ['build', '--tag=gcr.io/$PROJECT_ID/dataproc-spark-bigquery-connector-nightly', '-f', 'cloudbuild/Dockerfile', '.']
+
+# 2. Fetch maven and dependencies
+  - name: 'gcr.io/$PROJECT_ID/dataproc-spark-bigquery-connector-nightly'
+    id: 'init'
+    waitFor: ['docker-build']
+    entrypoint: 'bash'
+    args: ['/workspace/cloudbuild/presubmit.sh', 'init']
+    env:
+    - 'CODECOV_TOKEN=${_CODECOV_TOKEN}'
+
+# 3. Run unit tests
+  - name: 'gcr.io/$PROJECT_ID/dataproc-spark-bigquery-connector-nightly'
+    id: 'unit-tests'
+    waitFor: ['init']
+    entrypoint: 'bash'
+    args: ['/workspace/cloudbuild/presubmit.sh', 'unittest']
+    env:
+    - 'CODECOV_TOKEN=${_CODECOV_TOKEN}'
+
+  # 4. Run integration tests concurrently with unit tests
```

Comment:
Is this comment correct? It says Run integration tests concurrently with unit tests but I see a waitFor block

Reply:
updated the nightly.yaml